### PR TITLE
feat: add @output() method to expose current tooltip status as boolean

### PR DESCRIPTION
**Pull request by timhecker**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

It introduces a very basic feature to emit as `@Output()` if the tooltip is opened or not. I added `(handleShowing)="handleStatus($event)"` in the custom component block of playground and it works.

Issue Number: #11

## Does this PR introduce a breaking change?

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: projects/ngneat/helipopper/src/lib/helipopper.directive.ts [hunk 1]
```diff
@@ -118,6 +118,8 @@ export class HelipopperDirective implements OnDestroy {
   }
 
   @Output() helipopperClose = new Subject();
+  @Output() helipopperShowing = new Subject<boolean>();
```
Comment (NetanelBasal) [inline projects/ngneat/helipopper/src/lib/helipopper.directive.ts:121]: Can we change it to `helipopperVisible` please

File: README.md [hunk 2]
```diff
@@ -79,6 +79,22 @@ Now you can use it:
 </button>
 ```
 
+#### Handle current status as boolean
```
Comment (NetanelBasal) [inline README.md:82]: Just add it to the table of outputs

Comment (timhecker) [reply README.md:82]: Sorry, I can't find it: I found only Inputs table...

File: projects/ngneat/helipopper/src/lib/helipopper.directive.ts (hunk 1 above)
Comment (timhecker) [reply projects/ngneat/helipopper/src/lib/helipopper.directive.ts:121]: Done, looking forward for the next message about readme.

File: README.md (hunk 2 above)
Comment (NetanelBasal) [reply README.md:82]: Add one 😀

Comment (timhecker) [reply README.md:82]: Created new Outputs table. Let me know.

**Pull request merged**